Rakefile

```diff
@@ -2,7 +2,27 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'solr_wrapper/rake_task' unless Rails.env.production?
+require 'rspec/core/rake_task'
 
 Rails.application.load_tasks
 
-require 'solr_wrapper/rake_task' unless Rails.env.production?
+Rake::Task["test"].clear
+
+desc 'Execute the test build that runs on travis'
+task :test => [:environment] do
+  require 'solr_wrapper'
+  if Rails.env.test?
+    Rake::Task['db:migrate'].invoke
+    SolrWrapper.wrap do |solr|
+      solr.with_collection(name: 'blacklight-core', dir: "#{Rails.root}/solr/conf") do
+        Rake::Task['geoblacklight:solr:seed'].invoke
+        Rake::Task['spec'].invoke
+      end
+    end
+  else
+    system('rake ci RAILS_ENV=test')
+  end
+end
+
+task :default => :test
```

Comment:
doesn't this mean that we can also just do `rake` to run tests? thus we could take out the `script` part of `.travis.yml` because it runs `rake` automatically...or does `bundle exec rake` do something special?